src/styles/blocks/stars.scss

```
@@ -0,0 +1,16 @@
+.stars--4 .stars__star:nth-child(-n+4) {
+  width: 16px;
+  height: 16px;
+  background-image: url("../images/star-active.svg");
+  background-repeat: no-repeat;
```

You should put here only the values that are changing
```suggestion
  background-image: url("../images/star-active.svg");
```